
---

# obstacle avoidance service Response
int32 goal_angle